source/features/show-names.tsx

```diff
@@ -69,21 +64,40 @@ async function init(): Promise<false | void> {
 			' ',
 		);
 	}
+
+	batchedUsernameElements.clear();
+}, {
+	maxWait: 10,
+});
+
+// The selector observer calls this function several times, but we want to batch them into a single GraphQL API call
```

Comment:
We'll have to use this pattern more. Visually:

```
selector match -> animationstart -> callback
selector match -> animationstart -> callback
selector match -> animationstart -> callback
selector match -> animationstart -> callback
single batchUpdateLinks execution
```


## Test

- https://github.com on multiple "page" loads
- https://github.com/refined-github/sandbox/pull/10 correctly only triggered one request with only `cheap-glitch`

<img width="565" alt="Screen Shot 6" src="https://user-images.githubusercontent.com/1402241/190651186-4a02751e-8cb0-46ba-874a-7465327c3323.png">